endpoints/getting-started/main.py

```diff
@@ -89,7 +89,7 @@ def auth_info_firebase():
 @app.errorhandler(http_client.INTERNAL_SERVER_ERROR)
 def unexpected_error(e):
     """Handle exceptions by returning swagger-compliant json."""
-    logging.exception("An error occured while processing the request.")
+    logging.exception("An error occurred while processing the request.")
```

Comment:
```suggestion
    logging.exception("An error occured while processing the request.")
```

Removing because this is fixed in #11332